bin/serve.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http_server/http_server.dart' show VirtualDirectory;
+import 'package:path/path.dart';
+
+final webroot = join(dirname(dirname(Platform.script.toFilePath())), 'build');
+
+Future<void> main() async {
+  final virDir = new VirtualDirectory(webroot);
+
+  // Set up a directory handler to serve index.html files.
+  virDir.allowDirectoryListing = true;
+  virDir.directoryHandler = (dir, request) {
+    final indexUri = new Uri.file(dir.path).resolve('index.html');
+    virDir.serveFile(new File(indexUri.toFilePath()), request);
+  };
+
+  // TODO(dantup): How to decide port?
+  final server = await HttpServer.bind('127.0.0.1', 8765);
+
+  virDir.serve(server);
+  printJson({'host': server.address.host, 'port': server.port});
```

Comment:
I would remove this code before checkin unless the intent is for consumer to parse the json written to stdout. In that case, you should indicate this isn't logging output and ensure that the HttpServer is not writing anything to stdout.

Reply:
It was for tooling to parse :)

Originally it said "Listening at http://localhost:port/". Should I make it have a `--machine` flag and switch between the text/json? We need to provide the URL somehow so both users and tools know where to go (I don't think we want this script to spawn a browser directly).